### PR TITLE
Migrate to latest sticker contract

### LINKED ISSUE #8578: Migrate stickers feature to mainnet contract
### User Story

As a user, I want to purchase stickers on mainnet using SNT!

### Description

*Type*: Feature

*Summary*: We need to set up to use the mainnet version of the contract once deployed.

#### Expected behavior

I can buy and send stickers in chat on Mainnet.

#### Actual behavior

Stickers are only available on Ropsten.

### PR DESCRIPTION
**Pull request by jeluard**
fixes #8578

### Summary

Migrated to latest sticker market contract. No new feature. 

status: ready

### CODE REVIEW

**Review by dmitryn — APPROVED** [review]

**Review by yenda — APPROVED** [review]

File: src/status_im/ethereum/contracts.cljs [hunk 1]
```diff
@@ -8,7 +8,14 @@
    :status/tribute-to-talk
    {:testnet "0xC61aa0287247a0398589a66fCD6146EC0F295432"}
    :status/stickers
-   {:testnet "0x39d16CdB56b5a6a89e1A397A13Fe48034694316E"}})
+   {:testnet "0x8cc272396be7583c65bee82cd7b743c69a87287d"
+    :mainnet ""}
+   :status/sticker-market
+   {:testnet "0x6CC7274aF9cE9572d22DFD8545Fb8c9C9Bcb48AD"
+    :mainnet ""}
+   :status/sticker-pack
```
Comment (flexsurfer) [inline src/status_im/ethereum/contracts.cljs:16]: could please add descriptions for each stickers contract? why do we have 3 now instead 1 ?

**Review by vitvly — APPROVED** [review]

File: src/status_im/ethereum/contracts.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/ethereum/contracts.cljs:16]: It's part of latest @3esmit changes. See https://notes.status.im/Q-sQmQbpTOOWCQcYiXtf5g#
AFAIK it allows easier upgrades.

**Comment by asemiankevich** [comment]
```
10:54:22  + set +x
10:54:22  + IN_CI_ENVIRONMENT=1
10:54:22  + nix-shell --run '
10:54:22        make prod-build-desktop &&       ./scripts/build-desktop.sh buildClojureScript
10:54:22      ' --pure --keep VERBOSE_LEVEL --keep LOCALE_ARCHIVE_2_27 --keep IN_CI_ENVIRONMENT --argstr target-os macos /Users/jenkins/workspace/status-react_prs_macos_PR-8606/shell.nix
10:54:37  git clean -qdxf -f ./index.desktop.js desktop/ && \
10:54:37  lein prod-build-desktop && \
10:54:37  node prepare-modules.js
10:54:40  Compiling ClojureScript...
10:54:53  Compiling ["index.desktop.js" "status-modules/cljs/i18n-raw.js" "status-modules/cljs/network-raw.js" "status-modules/cljs/extensions-raw.js"] from ["components/src" "react-native/src/cljsjs" "react-native/src/desktop" "src" "env/prod" "prod"]...
10:55:16  [31mWARNING[0m: Use of undeclared Var status-im.utils.handlers/register-handler-fx at line 13 /Users/jenkins/workspace/status-react_prs_macos_PR-8606/src/status_im/wallet/choose_recipient/core.cljs
10:55:16  Error encountered performing task 'cljsbuild' with profile(s): 'prod'
10:55:16  Subprocess failed
10:55:16  make: *** [Makefile:146: prod-build-desktop] Error 1
```

@jeluard MacOS is never built here, could you assist?

**Comment by jeluard** [comment]
I think it is an issue with the release branch? I didn't touch anything that might be relevant

**Comment by asemiankevich** [comment]
@jakubgs maybe you can assist?

**Comment by flexsurfer** [comment]
@yenda it seems this error appears randomly

**Comment by asemiankevich** [comment]
I can buy , install and send/receive stickers on Ropsten here. Label is missing for Ropsten sticker packs though. Not sure how it looks on Mainnet since we dont have any sticker packs available yet there.
image

**Comment by jeluard** [comment]
@asemiankevich  Label?

**Comment by asemiankevich** [comment]
> @asemiankevich Label?

yes, on the screenshot  you can see white dot instead of [STT] or [SNT] or anything. Near the amount. 

image

**Pull request closed without merging**